packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -4359,6 +4359,18 @@ describe('scope hoisting', function () {
       assert.deepEqual(output.default, obj);
     });
 
+    it('should add a default interop for a CJS module', async function () {
+      let b = await bundle(
+        path.join(
+          __dirname,
+          '/integration/scope-hoisting/commonjs/interop-commonjs/a.js',
```

Comment:
More specificity in the fixture name could help, too:
```suggestion
          '/integration/scope-hoisting/commonjs/interop-commonjs-hybrid-dynamic/a.js',
```